src/Units/Weapons/TankMGun.java

```diff
@@ -5,7 +5,7 @@
 
   public TankMGun()
   {
-    super(WeaponType.TANKMGUN, 9);
+    super(WeaponType.TANKMGUN);
```

Comment:
This... is not really in scope for fog. This is minor so I won't fight it, but in general, we should try to keep PRs focused.